bluesky/bluesky.go

```diff
@@ -117,6 +121,42 @@ func (c *Client) GetProfile(
 	return bsky.ActorGetProfile(ctx, c.xrpc, actor)
 }
 
+func (c *Client) GetHead(
+	ctx context.Context, actorDID string,
+) (cid.Cid, error) {
+	resp, err := atproto.SyncGetHead(ctx, c.xrpc, actorDID)
+	if err != nil {
+		return cid.Cid{}, err
+	}
+	out, err := cid.Parse(resp.Root)
+	if err != nil {
+		return cid.Cid{}, err
+	}
+	return out, nil
+}
+
+func (c *Client) GetRecord(
```

Comment:
Nit: We could use generics here and take the expected type, and perform the assertion here. It'll mean we can avoid having the switch statement at each callsite. I'm happy with this as is though.